app/views/events/show.html.erb

```diff
@@ -0,0 +1,69 @@
+<div class="container justify-content-center my-5 col-4">
+  <label class="fs-3"><strong>Event:</strong></label>
+  <table class="table table-bordered table-hover table-striped">
+    <tbody>
+      <tr>
+        <th class="col-2">Name</th>
+        <td><%= @event.name %></td>
+      </tr>
+      <tr>
+        <th class="col-2">Category</th>
+        <td><%= @event.category.name %></td>
+      </tr>    
+      <tr>
+        <th class="col-2">Description</th>
+        <td><%= @event.description %></td>
+      </tr>
+      <tr>
+        <th class="col-2">Date</th>
+        <td><%= @event.event_date %></td>
+      </tr>
+    </tbody>
+  </table>
+
+  <div class="my-4 d-flex justify-content-center">
+    <% if Enrollment.find_by(event_id: @event.id, created: true).user_id == @current_user.id %>
+      <%= link_to "Edit", edit_event_path(@event), :class => 'btn btn-warning px-4' %>
+      <%= link_to "Delete", event_path(@event), :class => 'btn btn-danger ms-3', data: {
+        turbo_method: :delete,
+        turbo_confirm: :"Are You Sure?"
+      } %>
+      <%= link_to "All", events_path, :class => 'btn btn-info ms-3 px-4' %>
+    <% else %>   
+      <%= link_to "All", events_path, :class => 'btn btn-info ms-3 px-4' %>
+    <% end %>
+  </div>
+</div>
+
+<div class="container justify-content-center my-5 col-4">
+  <label class="fs-3"><strong>Comments:</strong></label>
+  <div class="border px-3 py-2">
+    <% @comments.each do |comment| %>
+      <div class="d-flex justify-content-between">
+        <div class="mb-2">
+          <p class="mb-1"><strong>Username: </strong><%= comment.user.username %></p>
+          <p class="mb-1"><strong>Comment: </strong><%= comment.comment %></p>
+          <p class="mb-1"><strong>Likes: </strong><%= comment.likes.count %></p>
+        </div>
+
+        <div class="mb-2 mt-4">
+          <% if comment.likes.pluck("user_id").include?(@current_user.id)%> 
```

Comment:
simplify this condiiton by doing a something like this :   comment.likes.where(user_id: @current_user.id).present?